netmaster/master/network.go

```diff
@@ -175,6 +176,45 @@ func deleteDockNet(tenantName, networkName, serviceName string) error {
 	return nil
 }
 
+// IsTagInRange verifies if the requested vlan/vxlan adheres to tenants configured range
+func IsTagInRange(pktTag int, pktTagType string, pktTagRange string, stateDriver core.StateDriver, tenantName string) bool {
```

Comment:
does this function need to be exported/public? Otherwise we can start it with lower-case.

Also can pktTag be negative? I am wondering if the `pktTag` argument to this function can be a uint instead to avoid the uint type cast below, otherwise we need to deal with testing input for negative numbers as well.